projectDocs/Design_Implementation.md

```diff
@@ -0,0 +1,54 @@
+# Design (Subject to updates)
+## Model  
+### Database  
+|   Database column   |   expected input   |        Thoughts on implementation        |
+| ------------------- | ------------------ | ---------------------------------------- |
+|          pk         |    auto-generate   |                                          |
+|       subject       |  string from user  | short description to user to add clarity |
+|  Issue comment FK   |  string from user  | long description to user to add clarity  |
+|     contactName     |  string from user  |                                          |
+|    dateSubmitted    |    auto-generate   |                                          |
+|     issueStatus     | Pulled from github |                                          |
+| github issue number | Pulled from github |                                          |
+|  github repository  | pulled from github |           probably don't need            |
```

Comment:
I think it's confusing when people read "probably don't need". I want this to be in the design, even though it will probably always have the same data value in it for the foreseeable future. The comment here might be that this is "over design" or allowing for future flexibility.